services/storage/src/simcore_service_storage/main.py

```diff
@@ -3,11 +3,10 @@
 import logging
 
 from servicelib.logging_utils import config_all_loggers
+from simcore_service_storage.core.application import create_app
```

Comment:
Absolute?

Reply:
I see 3 absolute imports here. Please check all of them

Reply:
they are necessary. otherwise it will not start. @giancarloromeo @GitHK have a look at all the main.py in your repository.